packages/web/src/components/PopupModal/PopupModal.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  useDisclosure,
+} from '@chakra-ui/react';
+
+interface PopupModalProps {
+  openModalText: String;
+  header: String;
+  body: String;
```

Comment:
Instead of having a body prop, we can use `children` so it'll be more flexible for non-text bodies

```suggestion
```